File: 171-ExcelColNum/excelColNum.java
public class Solution {
    public int titleToNumber(String s) {
        //power function: Math.pow(base, exponent)
        int rslt = 0;
         char[] arr = s.toCharArray();
         
         for (int i = 0; i< arr.length; i++){
             rslt += (arr[i] - 'A' + 1) * Math.pow(26, arr.length - i -1); 
         }
         return rslt;
        
    }
}

File: 171-ExcelColNum/excelColNum.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int res=0;
        int tmp=1;
        for(int i=s.size()-1;i>=0;i--){
            res+=(s[i]-'A'+1)*tmp;
            tmp*=26;
        }
        return res;
    }
};